instagram-app/src/components/CommentSection/CommentForm.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class CommentForm extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            value: ''
+        }
+    }
+
+    handleChange = event => {
+        this.setState({ value: event.target.value })
```

Comment:
To make this code reusable (say for a different input, in the future) - remember that the input 'name' can be used in setState as ```this.setState({ event.target.name: event.target.value })```